README.md

```diff
@@ -24,6 +24,7 @@ The **Code Insights** view keeps you informed about:
 - [Vulnerability](https://docs.datadoghq.com/security/application_security/vulnerability_management/) reports by Application Security Management
 - [Flaky tests](https://docs.datadoghq.com/continuous_integration/guides/flaky_test_management/) detected by CI Visibility
 - Profiling insights from [Watchdog Insights](https://docs.datadoghq.com/watchdog/insights/)
+- Log insights from [Log patterns](https://docs.datadoghq.com/logs/explorer/analytics/patterns/)
```

Comment:
There is some overlap between Code Insights and the log patterns that we are surfacing in the source editor, but we don't list the log insights in the Code Insights view, so is this really a correct description? 
![image](https://github.com/DataDog/datadog-for-intellij-platform/assets/1835893/8e84f334-7ad6-4633-9d12-4c4026b36842)